tests/integ/test_packaging.py

```diff
@@ -1010,7 +1010,6 @@ def import_scikit_fuzzy() -> str:
     assert "python-dateutil" in package_set
     assert "scikit-learn" in package_set
     assert "six" in package_set
-    assert "wrapt" in package_set
```

Comment:
Is this because of sktime 0.25.0 removed the dependency on wrapt?

Reply:
It's not very straightforward because I don't see 0.20.0 has `wrapt` either. I'm guessing this is from a nested dependency. 

Though frankly I don't think we should assert on non-direct dependencies because then our test is at the mercy of these packages.

Reply:
That's a great point 👍